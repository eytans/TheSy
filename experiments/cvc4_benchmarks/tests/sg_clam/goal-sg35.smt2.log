19:23:42 [WARN] Collecting statistics
19:23:42 [WARN]  ex is: zero
19:23:42 [WARN]  ex is: (succ zero)
19:23:42 [WARN]  ex is: (succ (succ zero))
19:23:42 [WARN] Running exploration without proof mode, but goals were given
19:23:42 [WARN] Proof mode: true
19:23:42 [WARN]  ex is: zero
19:23:42 [WARN]  ex is: (succ zero)
19:23:42 [WARN]  ex is: (succ (succ zero))
19:23:42 [WARN] Running exploration without proof mode, but goals were given
19:23:42 [WARN] Proof mode: true
19:23:42 [WARN] Found all lemmas
