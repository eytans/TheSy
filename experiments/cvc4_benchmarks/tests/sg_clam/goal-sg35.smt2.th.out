19:23:42 [0m[33m[WARN] [0mCollecting statistics
(leq ?__x0 ?__y1) => (or (= ?__x0 ?__y1) (less ?__x0 ?__y1))
(or (= ?__x0 ?__y1) (less ?__x0 ?__y1)) => (leq ?__x0 ?__y1)
(plus zero ?n) => ?n
(plus (succ ?n) ?m) => (succ (plus ?n ?m))
(succ (plus ?n ?m)) => (plus (succ ?n) ?m)
(mult zero ?n) => zero
(mult (succ ?n) ?m) => (plus (mult ?n ?m) ?m)
(plus (mult ?n ?m) ?m) => (mult (succ ?n) ?m)
(exp ?n zero) => (succ zero)
(exp ?n (succ ?m)) => (mult (exp ?n ?m) ?n)
(mult (exp ?n ?m) ?n) => (exp ?n (succ ?m))
(qexp ?n zero ?m) => ?m
(qexp ?n (succ ?m) ?p) => (qexp ?n ?m (mult ?p ?n))
(qexp ?n ?m (mult ?p ?n)) => (qexp ?n (succ ?m) ?p)
(mult (exp ?x ?y) ?z) => (qexp ?x ?y ?z)
(qexp ?x ?y ?z) => (mult (exp ?x ?y) ?z)
19:23:42 [0m[33m[WARN] [0m ex is: zero
19:23:42 [0m[33m[WARN] [0m ex is: (succ zero)
19:23:42 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:23:42 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:23:42 [0m[33m[WARN] [0mProof mode: true
19:23:42 [0m[33m[WARN] [0m ex is: zero
19:23:42 [0m[33m[WARN] [0m ex is: (succ zero)
19:23:42 [0m[33m[WARN] [0m ex is: (succ (succ zero))
19:23:42 [0m[33m[WARN] [0mRunning exploration without proof mode, but goals were given
19:23:42 [0m[33m[WARN] [0mProof mode: true
19:23:42 [0m[33m[WARN] [0mFound all lemmas
done in 1
